CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+cmake_minimum_required(VERSION 2.8)
+project( Gazo-san )
+set (CMAKE_CXX_STANDARD 11)
+set(GTEST OFF CACHE BOOL "Test flag")
+find_package( OpenCV REQUIRED )
+include_directories( ${OpenCV_INCLUDE_DIRS} )
+
+if(GTEST)
```

Comment:
Before build GTEST by cmake, we need running build_googletest.sh.
I think it is good to execute build_googletest.sh in this cmake if not exists google tests source.

Reply:
Add execute shell script in cmake.